gender_novels/corpus_gen.py

```diff
@@ -57,8 +57,16 @@ def generate_corpus_gutenberg():
         author = get_author_gutenberg(gutenberg_id)
         novel_metadata['author'] = author
         title = get_title_gutenberg(gutenberg_id)
+<<<<<<< HEAD
```

Comment:
There's a merge conflict here, indicated by `<<<<<<<< HEAD`, meaning that this file contains code from both your local repo and the upstream master. Please make sure to resolve it before committing again. (the easiest solution might be to just copy corpus_gen.py from the upstream master.